topologies/kne/ixia-cisco-ixia.textproto

```diff
@@ -0,0 +1,81 @@
+name: "ixia-c-2"
```

Comment:
Rename to cisco_ixia.textproto and make this 4 port?

Reply:
Yes sure.